runtime/sudate_test.ts

```diff
@@ -23,6 +23,11 @@ assert.equal(sud1.WeekDay(4), 0);
 assert.equal(sud3.MinusDays(sud1), 6);
 assert.equal(sud4.MinusSeconds(sud3), 12.345);
 
+// test daylight saving
+let sud7 = SuDate.make(2022, 10, 21, 0, 0, 0, 0);
+let sud8 = SuDate.make(2022, 11, 18, 0, 0, 0, 0);
+assert.equal(sud8.MinusDays(sud7), 28);
```

Comment:
You might want to have test cases that test two dates one second apart in the same day, versus two dates one second apart on different days, e.g.:

```
let sud9 = SuDate.make(2022, 10, 21, 0, 0, 0, 0);
let sud10 = SuDate.make(2022, 10, 21, 0, 0, 1, 0);
assert.equal(sud10.MinusDays(sud9), 0);

let sud11 = SuDate.make(2022, 10, 21, 23, 59, 59, 0);
let sud12 = SuDate.make(2022, 10, 22, 0, 0, 0, 0);
assert.equal(sud12.MinusDays(sud11), 1);
```

Also you should do all tests of `MinusDays` for both positive and negative results. Something like this:

```
assert.equal(sudx.MinusDays(sudy), z);
assert.equal(sudy.MinusDays(sudx), -z);
```

Also, my brother tells me some places change the hour at midnight, (Wikipedia suggests Chile is one such place), which may or may not complicate things, but would be worth testing.

Reply:
Sure, I will add more test cases. 

This test case is exactly for the hour change case. Nov. 6 is the end of the daylight saving of 2022, so one extra hour is added in the early morning. The previous code uses the local date to calculate the time difference which will be 28 days + 1 hour in ms. This will end up with the wrong result - 29 days (You have to run the tests in a time zone with daylight saving).